eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -301,10 +306,18 @@ stages:
               - ${{ each artifact in parameters.Artifacts }}:
                 - $(Pipeline.Workspace)/${{parameters.ArtifactName}}/PackageInfo/${{artifact.name}}.json
             WorkingDirectory: $(System.DefaultWorkingDirectory)
-            TargetDocRepoOwner: ${{parameters.TargetDocRepoOwner}}
+            TargetDocRepoOwner: Azure
             TargetDocRepoName: ${{parameters.TargetDocRepoName}}
             Language: 'dotnet'
             DailyDocsBuild: true
             SparseCheckoutPaths:
               - /api/overview/azure/
               - /metadata/
+        - ${{ each artifact in parameters.Artifacts }}:
```

Comment:
Instead of looping here one at a time can we please add support for batching inside of the script instead? As this causes us to duplicate a lot of steps in the pipeline.